acceptance_process.md

```diff
@@ -68,6 +40,7 @@ Use of resources is the question of how "expensive" a project is and its ability
 Risk control is about the question of how likely a project is to fail at delivering the expected result. Projects which are inherently more risky will be evaluated more stringently on their proposed risk control. Specific questions useful to the evaluation of risk control include:
 * How evenly is the effort spaced over the milestones of the project ?
 * How evenly is the payment spaced over the milestones of the project ?
+* How well did the applicant specify success criteria for the project ?
```

Comment:
Shouldn’t these ideally be templated or following guidance anyway?

Reply:
It's hard to template success criteria because they're different for every project. That said, the rewrite of the readme document should be helpful in this regard. Anyway the reason I added it here is to make it clear that lack of good metrics for success should be considered a risk driver (as well as hazard).